### PR TITLE
Bugfix/alert cancel appointment

### PR DESCRIPTION
**Pull request by evbambly**
image

* Alert before delete non-booked appointment
* Popup automatically closes on promise resolve

### CODE REVIEW

File: packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsScreen.tsx [hunk 1]
```diff
@@ -59,18 +59,27 @@ export default function ManageAppointmentsScreen({
       ),
     })
   );
+  const getPopupTitle = (
+    appointment: ManagedAppointment | undefined
+  ): string => {
+    if (!appointment?.booked) {
+      return "התור טרם אוכלס";
```
Comment (yaron1m) [inline packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsScreen.tsx:66]: התור טרם נתפס

File: packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsScreen.tsx [hunk 2]
```diff
@@ -59,18 +59,27 @@ export default function ManageAppointmentsScreen({
...
-        onApproved={() =>
-          popupData.appointment?.appointmentId &&
-          onDeleteAvailableAppointment(popupData.appointment?.appointmentId)
-        }
+        titleSecond={getPopupTitle(popupData.appointment)}
+        onApproved={async () => {
+          return await onDeleteAvailableAppointment(
+            popupData.appointment?.appointmentId || ""
+          );
+        }}
```
Comment (yaron1m) [inline packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsScreen.tsx:82]:
```
 onApproved={() => onDeleteAvailableAppointment(popupData.appointment?.appointmentId || "")}
```

There's no real need to specify this function is async

File: packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsScreen.tsx (hunk 1 above)
Comment (evbambly) [reply packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsScreen.tsx:66]: Done.

File: packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsScreen.tsx [hunk 3]
```diff
@@ -75,8 +75,8 @@ export default function ManageAppointmentsScreen({
         open={popupData.isOpen}
         titleFirst="האם ברצונך לבטל את התור?"
         titleSecond={getPopupTitle(popupData.appointment)}
-        onApproved={async () => {
-          return await onDeleteAvailableAppointment(
+        onApproved={() => {
+          return onDeleteAvailableAppointment(
```
Comment (yaron1m) [inline packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsScreen.tsx:79]: You can also remove the { and the return

**Review by yaron1m — APPROVED** [review]

**Pull request merged**